data/test_yaml_stories/stories_3.yml

```diff
@@ -0,0 +1,10 @@
+stories:
+- story: story with or and entities
```

Comment:
should we give the files names which reflect what they test instead of the the trailing numbers? E.g. `test_stories_with_or_and_entities.yml` ?